metadata/src/main/java/org/apache/kafka/metadata/authorizer/StandardAuthorizer.java

```diff
@@ -47,7 +54,7 @@
  * The standard authorizer which is used in KRaft-based clusters if no other authorizer is
  * configured.
```

Comment:
We can remove KRaft-related keywords to improve this document. because we only have KRaft based cluster in 4.X.